Numbers/OptionsMaker.cs

```diff
@@ -50,7 +50,10 @@ private static IEnumerable<PawnColumnDef> MainTable
             => new[] { DefDatabase<PawnColumnDef>.GetNamedSilentFail("Numbers_Inspiration"),
                          DefDatabase<PawnColumnDef>.GetNamedSilentFail("Numbers_Inventory"),
                          DefDatabase<PawnColumnDef>.GetNamedSilentFail("Numbers_SelfTend"),
-                         DefDatabase<PawnColumnDef>.GetNamedSilentFail("Numbers_Meditation"), }
+                         DefDatabase<PawnColumnDef>.GetNamedSilentFail("Numbers_Meditation"),
+                         DefDatabase<PawnColumnDef>.GetNamedSilentFail("Numbers_Psyfocus"),
+                         DefDatabase<PawnColumnDef>.GetNamedSilentFail("Numbers_Entropy"),
+                         DefDatabase<PawnColumnDef>.GetNamedSilentFail("Numbers_PsylinkLevel") }
```

Comment:
These 3/4 (including Meditation? not new) should maybe also be made conditional on Royalty being installed, but this is a static literal outside a class, and I'm not sure how to elegantly put conditionals here in C#.

Reply:
Should we include `Numbers_Meditation` (list focus types for pawn) as `DefModExtension_NeedsRoyalty` ? It was here before me, genuinely not sure (from game mechanics perspective) if it serves any purpose without Royalty.

Reply:
I'll change that on master then :+1: